traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -822,6 +825,7 @@ func updateV30(w http.ResponseWriter, r *http.Request, inf *api.APIInfo, reqDS *
 		&ds.RegionalGeoBlocking,
 		&ds.RemapText,
 		&ds.RoutingName,
+		&ds.ServiceCategoryId,
```

Comment:
this line should be moved to the end of the list right above `&ds.ID` to match the updateDSQuery()